lib/src/models/item_model.dart

```diff
@@ -0,0 +1,87 @@
+class ItemModel {
+  int _page;
+  List<_Result> _results = [];
+
+  ItemModel.fromJson(Map<String, dynamic> parsedJson) {
+    print(parsedJson['results'].length);
+    _page = parsedJson['page'];
+    List<_Result> temp = [];
+    for (int i = 0; i < parsedJson['results'].length; i++) {
+      _Result result = _Result(parsedJson['results'][i]);
+      temp.add(result);
+    }
+    _results = temp;
+  }
+
+  List<_Result> get results => _results;
+
+  int get page => _page;
+}
+
+class _Result {
+  List<int> _genreIds = [];
+  String _originalLanguage;
+  String _originalTitle;
+  String _posterPath;
+  bool _video;
+  double _voteAverage;
+  int _id;
+  String _overview;
+  String _releaseDate;
+  int _voteCount;
+  String _title;
+  bool _adult;
+  String _backdropPath;
+  double _popularity;
+  String _mediaType;
+
+  _Result(result) {
+    for (int i = 0; i < result['genre_ids'].length; i++) {
+      _genreIds.add(result['genre_ids'][i]);
+    }
+    _originalLanguage = result['original_language'];
+    _originalTitle = result['original_title'];
+    _posterPath = 'https://image.tmdb.org/t/p/w185' + result['poster_path'];
```

Comment:
Lets get this base_url as a constant in a Constants file 💯

Reply:
Added a new class that contains all the url called base_url